runtime/gc_check/CheckCycle.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include <string.h>
 
+#include "j9cfg.h"
```

Comment:
This belongs in `vendor_version.h` if it depends on things defined by `j9cfg.h`.

Reply:
Makes sense, updated.

Reply:
Updated again to fix it properly.